src/script-runner.js

```diff
@@ -34,26 +91,47 @@ window.web3Provider = {
 }
 window.web3 = new Web3(window.web3Provider)
 
-window.starknet = starknet
-
-console.log = function () {
-  window.remix.emit('log', {
-    data: Array.from(arguments)
+// 'getContractFactory' is added to 'ethers' to support usage of hardhat.ethers in the tests
+window.ethers.getContractFactory = (contractName, options) => {
```

Comment:
all the other functions should also be added see https://hardhat.org/plugins/nomiclabs-hardhat-ethers.html#helpers
could you also put all these in a separate file?

Reply:
will do this in a separate PR. Issue created: https://github.com/yann300/remix-script-runner/issues/23